x-pack/plugins/ml/server/types.ts

```diff
@@ -24,16 +24,16 @@ export interface LicenseCheckResult {
 
 export interface SystemRouteDeps {
   cloud: CloudSetup;
-  spacesPlugin: SpacesPluginSetup;
+  spacesPlugin?: SpacesPluginSetup;
```

Comment:
i just noticed that we use `spacesPlugin` here and `spaces` in `PluginsSetup` to refer to `SpacesPluginSetup`
but we use `cloud` in both to refer to `CloudSetup`.
It would be nice to have consistent naming for all plugin dependencies.

Reply:
but i'm not suggesting we change it in this PR. it's a bit out of scope

Reply:
It was a small change as it's only used in one of the route files. Updated here: 6fdc3375d75d9381b53e972c7a484d13b3dfc514